stdlib/2/cProfile.pyi

```diff
@@ -0,0 +1,17 @@
+from typing import Any, Callable, Dict, Optional, TypeVar
+
+def run(statement: str, filename: Optional[str] = ..., sort: int = ...) -> None: ...
+def runctx(statement: str, globals: Dict[str, Any], locals: Dict[str, Any], filename: Optional[str] = ..., sort: int = ...) -> None: ...
+
+SelfT = TypeVar('SelfT', bound='Profile')
```

Comment:
Add underscore prefix (e.g. `_SelfT`) so that this won't be imported when doing `from cProfile import *`.